services/notification/notifiers/tests/unit/test_comment.py

```diff
@@ -800,6 +800,63 @@ async def test_create_message_files_section_with_critical_files(
         )
         assert mocked_search_files_for_critical_changes.call_count == 2
 
+    @pytest.mark.asyncio
+    async def test_create_message_with_github_app_comment(
+        self,
+        dbsession,
+        mock_configuration,
+        mock_repo_provider,
+        sample_comparison,
+        mocker,
+    ):
+        comparison = sample_comparison
+        mocker.patch.object(
```

Comment:
This might be unecessary in this test if you only care for the 1st line of the comment